lib/msf/core/exploit/cmdstager/http.rb

```diff
@@ -4,7 +4,7 @@
 require 'msf/core/exploit/http/server'
 
 module Msf::Exploit::CmdStager
-module Http
```

Comment:
`HttpClient` and `HttpServer` are still named poorly, but I ain't changing those.